lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -169,9 +169,7 @@ function dosomething_reportback_files_form($form, &$form_state, $status = NULL,
 
   foreach ($result as $record) {
     $rb_path = 'admin/reportback/' . $record->rbid;
-    // @TODO: temporarily disable pre-approved to unblock campaigns team from reviewing reportbacks.
-    // Refs #4797
-    // $status = 'approved';
+    $status = 'approved';
```

Comment:
If we re-approve everything, we come back to the issue of user's submitting an RB and it getting auto-approved into the reviewed listing.